BasicSample/Assets/Shared/Scripts/RuntimeInfo.cs

```diff
@@ -59,6 +73,68 @@ private static string GetDisplayInfo()
             }
         }
 
+        private static string GetTrackingStates()
+        {
+            return $"Head: {GetTrackingState(XRNode.Head)} Left Hand: {GetTrackingState(XRNode.LeftHand)} Right Hand: {GetTrackingState(XRNode.RightHand)}";
+        }
+
+        private static string GetTrackingState(XRNode xRNode)
+        {
+            var trackingState = "Not found";
+            var inputDevice = InputDevices.GetDeviceAtXRNode(xRNode);
+            if (inputDevice.isValid && inputDevice.TryGetFeatureValue(CommonUsages.isTracked, out bool tracked))
+            {
+                trackingState = tracked ? "Tracked" : "Not tracked";
+            }
+
+            return trackingState;
+        }
+
+        private static string GetTrackingOriginMode()
+        {
+            XRInputSubsystem inputSubsystem = GetXRInputSubsystem();
+            if (inputSubsystem == null)
+            {
+                return "Tracking origin mode: Unknown";
+            }
+            else
+            {
+                return $"Tracking origin mode: {inputSubsystem.GetTrackingOriginMode()}";
+            }
+        }
+
+        private static XRInputSubsystem GetXRInputSubsystem()
+        {
+            XRGeneralSettings xrSettings = XRGeneralSettings.Instance;
+            if (xrSettings == null)
+            {
+                Debug.LogWarning($"GetXRInputSubsystem: XRGeneralSettings is null.");
+                return null;
+            }
+
+            XRManagerSettings xrManager = xrSettings.Manager;
+            if (xrManager == null)
+            {
+                Debug.LogWarning($"GetXRInputSubsystem: XRManagerSettings is null.");
+                return null;
+            }
+
+            XRLoader xrLoader = xrManager.activeLoader;
+            if (xrLoader == null)
+            {
+                Debug.LogWarning($"GetXRInputSubsystem: XRLoader is null.");
+                return null;
+            }
+
+            XRInputSubsystem xrInputSubsystem = xrLoader.GetLoadedSubsystem<XRInputSubsystem>();
```

Comment:
We should cache this subsystem until it stops running or something, so we aren't going through this whole null check and api call sequence every frame. The input subsystem shouldn't change often!

Reply:
Could you please clarify how to cache? Should i store it as a member variable and check for null and running before going through this? Also, note that this PR is merged so i can make these changes in a subsequent PR.

Reply:
I am also wondering, if instead of going via the loader, we can just get the input subsystem similar to the display subsystem?
something like:
var inputs = new List<XRInputSubsystem>();
SubsystemManager.GetInstances(inputs);
Would that work and if so, can i do this instead in Update ?